README.md

```diff
@@ -113,12 +113,12 @@ So it might be needed to guard certain features / workaround by the available se
 
 ## Releasing
 
-1. Run `bumpversion release`
-1. Generate the changelog (eg `towncrier --yes --version 0.1.0`)
-1. Commit your local changes, open a PR, and get it merged.
+1. Generate the changelog (eg `towncrier --yes --version 0.1.0`) and commit
```

Comment:
With this parameter towncrier is obviously not guessing the version, but could it?